streaming/index.js

```diff
@@ -271,6 +266,7 @@ if (cluster.isMaster) {
 
             transmit();
           }).catch(err => {
+            done();
```

Comment:
This might have contributed to problems too, although I doubt this was the main culprit, now